src/linalg/rankUpdate.jl

```diff
@@ -91,7 +91,7 @@ end
 
 function rankUpdate!(
     C::HermOrSym{T,Diagonal{T,Vector{T}}}, A::StridedMatrix{T}, α, β
-) where {T,S}
+) where {T}
```

Comment:
Good catch on this and the following change.

Reply:
I received a warning on Julia 1.8.2 when I loaded MixedModels!